models/PartnerBillingModel.v1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "PartnerBillingModel",
+  "type": "object",
+  "description": "Modèle de facturation du partenaire. \n\nSi from_amount et to_amount ne sont pas remplis, le modèle de facturation s'appliquera sur l'intégralité du montant du prêt, comme: fix_part + montant_du_prêt * variable_part\n\nSinon, la facturation sera calculée comme: fix_part + (to_amount - from_amount) * variable_part\n\n",
```

Comment:
Maybe this comment needs to be changed to reflect the new names of fields.
In addition the last formula seems wrong? 
Shouldn't it be: 
```
flat_fee + (loan_amount - from_amount) * rate 
```

Reply:
Indeed, I updated the comment.

Regarding the formula, I don't think it's wrong: with your proposal, the part between from_amount and loan_amount will be counted as many times as we have tiers. In fact, your formula is the one for the highest tier. 

What I propose is the same approach as for the French "impots sur le revenus".

Reply:
Ok I see so basically it is:

```
flat_fee + (min(loan_amount, to_amount) - from_amount) * rate
```

right?